api/lib/infrastructure/repositories/competence-tree-repository.js

```diff
@@ -2,7 +2,7 @@ const areaRepository = require('./area-repository');
 const CompetenceTree = require('../../domain/models/CompetenceTree');
 
 module.exports = {
-  get() {
-    return areaRepository.listWithPixCompetencesOnly().then((areas) => new CompetenceTree({ areas }));
+  get({ locale } = {}) {
+    return areaRepository.listWithPixCompetencesOnly({ locale }).then((areas) => new CompetenceTree({ areas }));
```

Comment:
nitpick: passer en async/await
```suggestion 
  async get({ locale } = {}) {
   const areas = await areaRepository.listWithPixCompetencesOnly({ locale });
   return new CompetenceTree({ areas }));
```